### ISSUE TITLE
Error on interpolation

### ISSUE THREAD

**Issue by BaGsn** [byte-foundry/plumin.js#13]
When I try the interpolation demo, I have this error

`glyph.updateOTCommands is not a function.`

It send me to this part of the code (line 20650)
```
	this.ot.glyphs.glyphs = (
		this.getGlyphSubset( args && args.set ).reduce(function(o, glyph, i) {
			o[i] = args && args.shouldUpdateCommands ?
				glyph.updateOTCommands( null, args && args.merged ) :
				glyph.ot;
			return o;
		}, {})
	);
```

**Comment by louisremi** [comment]
I'm looking into it right now.

**Comment by louisremi** [comment]
Fixed in 1.0.0 : )
